vspec.py

```diff
@@ -212,6 +214,23 @@ def load(file_name, include_paths):
     return deep_model["children"]
 
 
+def convert_yaml_to_list(raw_yaml):
+    if isinstance(raw_yaml, list):
+        return raw_yaml
+
+    # Sort the dictionary according to line number.
+    # The reason is that when the YAML file is loaded
+    # the object order is not preserved in the created
+    # dictionary
+    raw_yaml = collections.OrderedDict(sorted(raw_yaml.items(), key=lambda x: x[1]['$line$']))
```

Comment:
Here (and even more elsewhere) "line_number" might a bit more understandable than "line"